<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AskAIChat</class>
 <widget class="QWidget" name="AskAIChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ask AI Chat</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">#AskAIChat{	
	
	border: 1px solid #274b6f;
	border-radius: 5px;
	padding: 5px 10px;
	color: rgb(40, 73, 107);
	background-color: rgb(249, 249, 249);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="label_title">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string>
       font-size: 18px;
       font-weight: bold;
       color: rgb(40, 75, 107);
      </string>
     </property>
     <property name="text">
      <string>Ask AI Chat</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="text_input">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
	border: 1px solid #274b6f;
	border-radius: 5px;
	padding: 5px 10px;
	color: rgb(40, 73, 107);
	background-color: rgb(249, 249, 249);
</string>
     </property>
     <property name="placeholderText">
      <string>Type your question here...</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="button_send">
        <property name="styleSheet">
         <string notr="true">
       background-color: rgb(40, 75, 107);
       color: white;
       padding: 5px;
	border-radius: 5px;
      </string>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_clear">
        <property name="font">
         <font/>
        </property>
        <property name="styleSheet">
         <string notr="true">
       background-color: rgb(228, 60, 76);
       color: white;
       padding: 5px;
	border-radius: 5px;
      </string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_loading">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string>
       font-size: 14px;
       color: rgb(150, 150, 150);
      </string>
     </property>
     <property name="text">
      <string>Your answer is on the way...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scroll_area">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scroll_area_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>987</width>
        <height>216</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <layout class="QVBoxLayout" name="scroll_area_layout">
       <item>
        <widget class="QLabel" name="label_answer">
         <property name="styleSheet">
          <string>
           font-size: 14px;
           color: rgb(40, 75, 107);
          </string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
